Dockerfile

```diff
@@ -17,24 +19,28 @@ ENV DOTNET_SKIP_FIRST_TIME_EXPERIENCE 1
 RUN apt-get update \
     && apt-get install -y --no-install-recommends \
         libc6 \
-        libcurl3 \
         libgcc1 \
         libgssapi-krb5-2 \
-        libicu57 \
-        liblttng-ust0 \
-        libssl1.0.2 \
+        libicu66 \
+        libssl1.1 \
         libstdc++6 \
         libunwind8 \
-        libuuid1 \
         zlib1g \
     && rm -rf /var/lib/apt/lists/*
 
-# Install .NET Core SDK
-ENV DOTNET_SDK_VERSION 2.0.0
-ENV DOTNET_SDK_DOWNLOAD_URL https://dotnetcli.blob.core.windows.net/dotnet/Sdk/$DOTNET_SDK_VERSION/dotnet-sdk-$DOTNET_SDK_VERSION-linux-x64.tar.gz
-ENV DOTNET_SDK_DOWNLOAD_SHA E457F3A5685382F7F24851A2E76EDBE75B575948C8A7F43220F159BA29C329A5008BBE7220C18DFB31EAF0398FC72177B1948B65E19B34ED0D907EFB459CF4B0
-
-RUN curl -SL $DOTNET_SDK_DOWNLOAD_URL --output dotnet.tar.gz \
+RUN if [ "${TARGETPLATFORM}" = "linux/arm64" ] ; then \
+        DOTNET_SDK_VERSION=3.1.419; \
```

Comment:
The [dotnet SDK v2](https://dotnet.microsoft.com/en-us/download/dotnet/2.0) does not have a arm64 binary, and the oldest dotnet SDK version with an arm64 binary is v3. I went with 3.1.419 because that was documented in [docker-dotnet](https://github.com/dotnet/dotnet-docker/blob/main/src/sdk/3.1/focal/arm64v8/Dockerfile#L28).  Also, v3.1 is the oldest [ubuntu v20.04 supported version of dotnet](https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#supported-distributions).  We are using ubuntu-20.04 [as the github actions runner](https://github.com/Dwolla/jenkins-agent-docker-dotnet/blob/multi-arch-builds/.github/workflows/ci.yml#L14).

Would it make sense to use dotnet sdk v3.1.419 for amd64 as well? Will that break processes using this agent downstream that might be depending on dotnet v2?